### DISCUSSION TITLE
README.md の「Creating a Project」がリンク切れ

### DISCUSSION THREAD

**Discussion by Acha0203** [sinProject-Inc/talk#1385] [Q&A]
README.md の「Creating a Project」がリンク切れになっています。
環境構築はどうにかできましたが、「Creating a Project」のリンク先に正しい環境構築手順が書かれているのでしたら、ぜひ拝読したいのですが。

**Comment by sinproject-iwasaki** [comment]
@Acha0203 

リンク切れ報告ありがとうございます。
こちらになります。

https://sinpro.dev/docs/creating-a-project

**Reply by Acha0203** [reply]
ご返信ありがとうございます。リンク先を確認いたしました。
